### ISSUE TITLE
Support for Chosen or any other custom element

### ISSUE THREAD

**Issue by Alexwijn** [claviska/jquery-dropdown#77]
I'm using Chosen inside your dropdown element and because of that I need to adjust the script to avoid that the dropdown box is closing when I click it. This is because chosen is an "A" element.

I add the following:
```if (targetGroup.is('.chosen-container')) { return; }```

In the following if-statement:
```if (targetGroup && targetGroup.is('.dropdown')) {```

Any change you could add support things like this? For example have a config variable where you can specify specific elements that will not close it.

Thanks.

**Comment by claviska** [comment]
Makes sense. I'm open to a PR because I probably won't get to it right away :)

**Comment by rythos42** [comment]
I just updated my copy for my purposes, but I wanted the opposite -- to close the drop down when clicking an INPUT (button) element.
